M/msolve/build_tarballs.jl

```diff
@@ -42,8 +45,8 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("GMP_jll", v"6.2.0"),
-    Dependency("FLINT_jll", compat = "~300.100.301"),
+    Dependency("GMP_jll", v"6.2.1"),
+    Dependency("FLINT_jll", compat = "~300.200.0"),
```

Comment:
```suggestion
    Dependency("FLINT_jll", compat = "~300.100.301"),
```
I would suggest to build `msolve_jll v0.700.500` still with FLINT v3.1.3.
There are some complications in getting new normaliz and polymake builds with FLINT v3.2.0, so that could delay everything a bit. And in particular, Oscar will not use any msolve build with FLINT v3.2.0 until *all* dependencies are rebuild.
I can then provide a `msolve_jll v0.700.501` or similar (depending on how long the rest takes) that builds against FLINT v3.2.0